tokio-native-tls/Cargo.toml

```diff
@@ -30,10 +30,10 @@ vendored = ["native-tls/vendored"]
 
 [dev-dependencies]
 tokio = { version = "1.0", features = ["macros", "rt", "rt-multi-thread", "io-util", "net"] }
-tokio-util = { version = "0.6.0", features = ["full"] }
+tokio-util = { version = "0.7.4", features = ["full"] }
```

Comment:
Any particular reason to go with .4 here? Coming from 0.6.x we presumably by default don't need any new features introduced after 0.7.0.

Reply:
no reason. happy to do 0.7.0 if you prefer

Reply:
Stood out to me. But given that 0.7.4 has been out for a while, seems fine.